### PR TITLE
:bug:(lld): fix toasts issue

### PR DESCRIPTION
**Pull request by LucasWerey**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - toasts

### 📝 Description

Stacked were not disappearing because we didn't delete their id of the list. Fixed ✅ 
Also add duration to toast of 10sec
Fixed the duration bar 

https://github.com/user-attachments/assets/8fc1a9ca-59ea-4940-bfe3-59302e3e8433

### ❓ Context

- **JIRA or GitHub link**:(https://ledgerhq.atlassian.net/browse/LIVE-14262)[LIVE-14262] 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-14262]: https://ledgerhq.atlassian.net/browse/LIVE-14262?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by cgrellard-ledger — APPROVED** [review]

File: apps/ledger-live-desktop/src/renderer/components/ToastOverlay/TimeBasedProgressBar.tsx [hunk 1]
```diff
@@ -29,25 +22,22 @@ export function TimeBasedProgressBar({ duration, onComplete, nonce = 1 }: Props)
...
-            background: "#8a80db", // "#00000022",
-          }}
-        />
-      ))}
+      <animated.div
+        style={{
+          ...style,
+          height: 5,
+          transformOrigin: "left center",
+          background: "#8a80db",
```
Comment (themooneer) [inline apps/ledger-live-desktop/src/renderer/components/ToastOverlay/TimeBasedProgressBar.tsx:38]: you may need to use theme colors to handle dark/light mode.
This is a suggestion as i don't know if the toast need to be white on both cases.

**Review by themooneer — APPROVED** [review]

File: apps/ledger-live-desktop/src/renderer/components/ToastOverlay/TimeBasedProgressBar.tsx (hunk 1 above)
Comment (LucasWerey) [reply apps/ledger-live-desktop/src/renderer/components/ToastOverlay/TimeBasedProgressBar.tsx:38]: This colour is not in our DS so I assume it was a decision when the component has been created to have it like this

Comment (cgrellard-ledger) [reply apps/ledger-live-desktop/src/renderer/components/ToastOverlay/TimeBasedProgressBar.tsx:38]: Indeed I think it was the case Lucas, it currently looks okay in both themes so it's okay to keep this one like that but we won't do it this way for any new features

Comment (themooneer) [reply apps/ledger-live-desktop/src/renderer/components/ToastOverlay/TimeBasedProgressBar.tsx:38]: sounds clear for me, thank you both

**Pull request merged**